src/renderer/shared/lib/utils/substrate.ts

```diff
@@ -3,8 +3,10 @@ import { BaseTxInfo, getRegistry, GetRegistryOpts, OptionsWithMeta, TypeRegistry
 import { isHex, hexToU8a, bnMin, BN_TWO, BN } from '@polkadot/util';
 import { blake2AsHex } from '@polkadot/util-crypto';
 
-import { Address, CallData, CallHash } from '@shared/core';
+import { Address, CallData, CallHash, XcmPallets } from '@shared/core';
+// TODO: Incorrect import
```

Comment:
All of these constants are used only in `shared/utils/substrate` we can safely move to `shared/utils/constants`
```javascript
export const THRESHOLD = BN_THOUSAND.div(BN_TWO);
export const DEFAULT_TIME = new BN(6_000);
export const ONE_DAY = new BN(24 * 60 * 60 * 1000);
```